developer/js/source/lexical-model-compiler/build-trie.ts

```diff
@@ -380,31 +383,35 @@ namespace Trie {
         .reduce((acc, count) => acc + count, 0);
     }
   }
+}
 
-  /**
-   * Converts word forms in into an indexable form. It does this by converting
-   * the string to uppercase and trying to remove diacritical marks.
-   *
-   * This is a very naïve implementation, that I've only though to work on
-   * languages that use the Latin script. Even then, some Latin-based
-   * orthographies use code points that, under NFD normalization, do NOT
-   * decompose into an ASCII letter and a combining diacritical mark (e.g.,
-   * SENĆOŦEN).
-   *
-   * Use this only in early iterations of the model. For a production lexical
-   * model, you SHOULD write/generate your own key function, tailored to your
-   * language.
-   */
-  function defaultWordform2Key(wordform: string): SearchKey {
-    // Use this pattern to remove common diacritical marks.
-    // See: https://www.compart.com/en/unicode/block/U+0300
-    const COMBINING_DIACRITICAL_MARKS = /[\u0300-\u036f]/g;
-    return wordform
-      .normalize('NFD')
-      .toLowerCase()
-      // remove diacritical marks.
-      .replace(COMBINING_DIACRITICAL_MARKS, '') as SearchKey;
-  }
+/**
+ * Converts wordforms into an indexable form. It does this by
+ * normalizing the letter case of Latin characters and removing
+ * common diacritical marks.
+ *
+ * This is a very naïve implementation, that I only think will work on
+ * some language that use the Latin script. As of 2020-04-08, only
```

Comment:
```suggestion
 * some languages that use the Latin script. As of 2020-04-08, only
```

(I note this comment is not char-for-char identical with the one in trie-model.ts 😉).